src/main/java/com/mraof/minestuck/alchemy/PredeterminedCardCaptchas.java

```diff
@@ -0,0 +1,20 @@
+package com.mraof.minestuck.alchemy;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+import net.minecraft.world.item.Item;
+
+
+public class PredeterminedCardCaptchas
+{
+	public static BiMap<Item, String> predefinedCardMap = HashBiMap.create();
```

Comment:
I get the impression that this field shouldn't really be public.
Try lowering the visibility and adding some functions that control how this map may be accessed.